WindowsCompatibility/WindowsCompatibility.psm1

```diff
@@ -698,7 +698,7 @@ function Add-WindowsPSModulePath
     $WindowsPSModulePath = [System.Environment]::GetEnvironmentVariable("psmodulepath", [System.EnvironmentVariableTarget]::Machine)
     if (-not ($env:PSModulePath).Contains($WindowsPSModulePath))
     {
-        $env:PSModulePath += ";${env:userprofile}\Documents\WindowsPowerShell\Modules;${env:programfiles}\WindowsPowerShell\Modules;${WindowsPSModulePath}"
+        $env:PSModulePath += ";${env:userprofile}\Documents\WindowsPowerShell\Modules;${WindowsPSModulePath}"
```

Comment:
I see merit in adding the path explicitly (environment somehow excludes this well known path from the env var). However, it may be better to only add the path if it is not already present.